src/lib/helperFunctions.ts

```diff
@@ -45,3 +45,24 @@ export const statusToNumber = (
   }
   return 0;
 };
+
+export const isHostnameLocal = (hostname: string) => {
+  if (hostname === "localhost" || hostname === "[::1]") {
+    return true;
+  }
+  if (hostname.startsWith("127.0.0.") && hostname.length <= 11) {
```

Comment:
Why not IPv6? `::1` is the only valid ip6 loopback address as far as I'm aware and since it's URI hostnames, the address surrounded by the square braces or am I getting you wrong?

Reply:
Changed test to support the whole range with 11667c0